### PR TITLE
Camera settings

### PR DESCRIPTION
**Pull request by AngieDutra**
## Motivation

Camera needed more setting options

## Summary of changes

Tis PR adds perspective settings:
- Default
- Zoom in
- Zoom out
- Sentinel 

## Checklist
- [x] I have tested the changes locally.
- [x] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

**Review by alfopisano — CHANGES_REQUESTED** [review]
Nice!! I just have a question, when on the sentinel camera I can not see the health bars, is that alright? 

https://github.com/lambdaclass/curse_of_mirra/assets/116388676/70d99b59-6119-44d0-87d2-2eae5b41f62f

**Comment by AngieDutra** [comment]
Our PM said he'll create an issue for this and we should move on if this is the only thing found so far

**Comment by alfopisano** [comment]
As discussed with the team, we should leve it as it is for now and approach it on future iterations.

**Review by alfopisano — DISMISSED** [review]

File: client/Assets/Scripts/UI/CameraManager.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+using UnityEngine.UI;
+using Cinemachine;
+
+public class CameraManager : MonoBehaviour
+{
+    [SerializeField] 
+    Camera mainCamera;
+
+    [SerializeField] 
+    GameObject mainCameraCM;
```
Comment (BertovDev) [inline client/Assets/Scripts/UI/CameraManager.cs:14]:
```
    [SerializeField] 
    CinemachineVirtualCamera mainCameraCM;
```

File: client/Assets/Scripts/UI/CameraManager.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    List<GameObject> cameraToggles;
+
+    private Vector3 defaultCameraRotation;
+    private Vector3 topView = new Vector3(90, 0, 0);
+    private float initialCameraDistance, variableCameraDistance;
+    
+    void Awake()
+    {
+        defaultCameraRotation = mainCamera.transform.rotation.eulerAngles;
+        initialCameraDistance = mainCameraCM.GetComponent<CinemachineVirtualCamera>().GetCinemachineComponent<CinemachineFramingTransposer>().m_CameraDistance;
```
Comment (BertovDev) [inline client/Assets/Scripts/UI/CameraManager.cs:26]:
```
        cinemachineFramingTransposer = mainCameraCM.GetCinemachineComponent<CinemachineFramingTransposer>();
        initialCameraDistance = cinemachineFramingTransposer.m_CameraDistance;
```

File: client/Assets/Scripts/UI/CameraManager.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+    public void SetSentinelSettings()
+    {
+        SetCamera(topView, initialCameraDistance, true);       
+        UpdateUIState(cameraToggles[3]); 
+    }
+    public void SetCamera(Vector3 cameraRotation, float cameraDistance, bool orthographic)
+    {
+        mainCameraCM.transform.rotation = Quaternion.Euler(cameraRotation);
+        mainCameraCM.GetComponent<CinemachineVirtualCamera>().GetCinemachineComponent<CinemachineFramingTransposer>().m_CameraDistance = cameraDistance;
```
Comment (BertovDev) [inline client/Assets/Scripts/UI/CameraManager.cs:52]: `       cinemachineFramingTransposer.m_CameraDistance = cameraDistance;`

**Review by BertovDev — CHANGES_REQUESTED** [review]
Nice working good and I am not a big fan of creating classes to only holding a reference. Instead I would do something like this. Wdyt?

**Comment by AngieDutra** [comment]
I'll try this then

**Comment by AngieDutra** [comment]
Everything was added or changed as solicited

**Review by BertovDev — APPROVED** [review]
LGTM

Tested on Android s23

**Review by alfopisano — APPROVED** [review]
Nice!! Tested on iOS build

**Pull request merged**